starport-core/src/main/scala/com/krux/starport/util/notification/EmailNotification.scala

```diff
@@ -0,0 +1,17 @@
+package com.krux.starport.util.notification
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.db.record.Pipeline
+
+protected class EmailNotification extends Notification {
+	override def send(summary: String, message: String, pipeline: Pipeline)(implicit conf: StarportSettings): String = {
```

Comment:
Let's use 2 space indentation, sent you our style guide offline, will work on open sourcing it so that we can start having links to in those open source projects soon

Reply:
Done